Wikipedia/Code/CLLocationManager+WMFLocationManagers.h

```diff
@@ -10,10 +10,17 @@
 + (instancetype)wmf_fineLocationManager;
 
 /**
- *  Used to update business logic whenever location changes significantly.
+ *  Used to update business logic whenever location changes 10 meters.
  *
  *  @return A location manager configured for coarse-grained updates.
  */
 + (instancetype)wmf_coarseLocationManager;
 
+
+/**
+ *  Used to update business logic whenever location changes significantly.
+ *
+ *  @return A location manager configured for coarse-grained updates.
+ */
++ (instancetype)wmf_significantLocationManager;
```

Comment:
Could we rename `wmf_coarseLocationManager` and `wmf_significantLocationManager` to more clearly state how they're different?